src/php/Compiler/Emitter/OutputEmitter/NodeEmitter/MapEmitter.php

```diff
@@ -16,10 +16,40 @@ public function emit(AbstractNode $node): void
     {
         assert($node instanceof MapNode);
 
-        $this->outputEmitter->emitContextPrefix($node->getEnv(), $node->getStartSourceLocation());
+        /*$this->outputEmitter->emitContextPrefix($node->getEnv(), $node->getStartSourceLocation());
         $this->outputEmitter->emitStr('\Phel\Lang\TypeFactory::getInstance()->persistentMapFromKVs(', $node->getStartSourceLocation());
         $this->outputEmitter->emitArgList($node->getKeyValues(), $node->getStartSourceLocation());
         $this->outputEmitter->emitStr(')', $node->getStartSourceLocation());
+        $this->outputEmitter->emitContextSuffix($node->getEnv(), $node->getStartSourceLocation());*/
```

Comment:
What about this?

Reply:
If this is not used, I would suggest removing it